job_controller/job_controller.go

```diff
@@ -48,6 +48,12 @@ type ControllerInterface interface {
 	// Returns the Group Name(value) in the labels of the job
 	GetGroupNameLabelValue() string
 
+	// Returns the Replica Type(key) in the labels of the job
+	GetReplicaTypeLabelKey() string
```

Comment:
Is this because of merge conflict? https://github.com/kubeflow/common/pull/29 removes these labels

Reply:
I think so. @merlintang Could you rebase?

Reply:
it is fixed. thanks for your suggestion.

Reply:
These interfaces are supposed to be removed and use [this](https://github.com/kubeflow/common/blob/39cbea8d8da2c633a706143f03fee442fa0c1f10/util/status.go#L22) instead